modules/post/linux/gather/enum_network.rb

```diff
@@ -113,7 +114,8 @@ def get_ssh_keys
     keys = []
 
     #Look for .ssh folder, "~/" might not work everytime
-    dirs = execute("/usr/bin/find / -maxdepth 3 -name .ssh").split("\n")
```

Comment:
I am not using `execute` method in this because now `execute` only returns the output if the command runs successfully but in this case even when some directories are readable by the user the method will return `nil` because of ` || echo {error_token}`